solidity/dashboard/src/sagas/liquidity-rewards.js

```diff
@@ -124,7 +124,7 @@ export function* watchLiquidityRewardNotifications() {
       )
     }
     displayMessage = true
-    yield delay(moment.duration(7, "minutes").asMilliseconds()) // every 7 minutes
+    yield delay(moment.duration(30, "minutes").asMilliseconds()) // every 7 minutes
```

Comment:
I think we can remove comment `// every 7 minutes`. `moment.duration()` is enough.